tidb.go

```diff
@@ -74,7 +73,9 @@ var (
 	domap = &domainMap{
 		domains: map[string]*domain.Domain{},
 	}
-	stores = make(map[string]kv.Driver)
+	stores    = make(map[string]kv.Driver)
```

Comment:
Since TiDB can be used as a library, having these global variable are inconvenient if not dangerous.